### ISSUE TITLE
warning issued when running medeas on test and through the snamake file

### ISSUE THREAD

**Issue by sapfo** [sapfo/medeas#32]
/home/sapfo/projects/mds/medeas/src/extrapolate_split_time.py:20: RuntimeWarning: invalid value encountered in true_divide
 all_contribution = (ts[:-1] + ns + delta_ts/(1-np.exp(delta_ts/ns)))*probability

not sure if that matters?

**Comment by frederic-michaud** [comment]
Yes, I'm aware of this warning. I know this does not impact the results, but I can try to remove it.
